examples/server/server.cpp

```diff
@@ -2368,6 +2368,17 @@ int main(int argc, char **argv)
                                     break;
                                 }
                             } else {
+                                const std::string str =
+                                "error: " +
+                                result.result_json.dump(-1, ' ', false, json::error_handler_t::replace) +
+                                "\n\n";
```

Comment:
I'm not a fan of the code style here, but it's consistent with the rest of this file so I suppose it's acceptable.